lua/entities/base_wire_entity.lua

```diff
@@ -324,28 +348,32 @@ if CLIENT then return end -- no more client
 --------------------------------------------------------------------------------
 
 util.AddNetworkString( "wire_overlay_data" )
+util.AddNetworkString( "wire_overlay_txt" )
+util.AddNetworkString( "wire_overlay_request" )
 
-timer.Create("WireOverlayUpdate", 0.1, 0, function()
-	for _, ply in ipairs(player.GetAll()) do
-		local ent = ply:GetEyeTrace().Entity
-		if IsValid(ent) and ent.IsWire and
-			ent.OverlayData and
-			ent.OverlayData_UpdateTime and
-			ent.OverlayData_UpdateTime.time > (ent.OverlayData_UpdateTime[ply] or 0) then
+ENT.playersRequestingOverlayNumeric = {}
```

Comment:
Move to `self.playersRequestingOverlayNumeric = {}` in Initialize

Reply:
this breaks it, it never gets created.

Reply:
If the initialize function doesn't get called we have bigger issues than that.